.github/workflows/resyntax-autofixer.yml

```diff
@@ -23,8 +23,11 @@ jobs:
           dest: '"${HOME}/racketdist-current-CS"'
           sudo: never
       - name: Install and setup
-        run: raco pkg install --auto gui-doc gui-lib gui-test gui text-table
+        run: |
+          raco pkg install -i --auto --no-setup --skip-installed gui-test
+          raco pkg update --auto --no-setup gui-doc gui-lib gui text-table
+          raco setup gui-doc gui-lib gui-test gui text-table
```

Comment:
you might want -D here

Reply:
I won't bother for now, since eventually I'd like Resyntax to work on scribble source files too. See jackfirth/resyntax#387 for an example use case.